tools/clang/test/CodeGenSPIRV_Lit/vk.layout.asbuffer.std430.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %dxc -T vs_6_0 -E main
+// RUN: %dxc -T vs_6_0 -E main -fvk-use-gl-layout -fcgl  %s -spirv | FileCheck %s
```

Comment:
This one passes on my end without the new layout option if I clone the PR and run lit:
`./build/bin/llvm-lit tools/clang/test/ --filter=CodeGenSPIRV_Lit/vk.layout.asbuffer -v`
(fetched your PR, so we should have the same base)